hazelcast/src/main/java/com/hazelcast/internal/jmx/ManagementService.java

```diff
@@ -101,10 +103,17 @@ public void destroy() {
         }
     }
 
-    public static void shutdownAll() {
+    public static void shutdownAll(List<HazelcastInstanceProxy> instances) {
+        for (HazelcastInstanceProxy instance : instances) {
+            shutdown(instance.getName());
```

Comment:
is it possible to return null from `instance.getName()`? What will be the impact in case of null return?